requirements.txt

```diff
@@ -0,0 +1,7 @@
+datajoint>=0.13.0
+element-animal
+element-lab
+element-session
+element-calcium-imaging
+element-zstack
+intern
```

Comment:
Please pin these versions with the `>=` notation.

Reply:
I will update this in the next commit